src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 0;
+  }
```

Comment:
According to the checklist, the image should be centered. Use `margin: 32px auto 0;` instead of `margin: 32px 19px 0;` to center the image horizontally within the card.